app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+
+import EmptyNotesListImage from "images/EmptyNotesList";
+import { PageLoader } from "neetoui";
+
+import EmptyState from "components/Common/EmptyState";
+import TabsHeadersTable from "components/Common/TabsHeadersTable";
+import { CONTACTS, SEGMENTS, TABS, TAGS } from "constants/contacts";
```

Comment:
Let's keep constants scoped to a feature. If the constants are being used across folders, then, it makes sense to add them to a global folder and use an alias. Here, we can place it inside `Contacts` folder.

Reply:
Do you mean that instead of `constants/contacts` it should be `contacts/constants`? And use it from there.

Reply:
Yup.